src/main/java/fr/insalyon/citi/golo/compiler/JavaBytecodeGenerationGoloIrVisitor.java

```diff
@@ -451,14 +454,26 @@ public void visitThrowStatement(ThrowStatement throwStatement) {
     methodVisitor.visitInsn(ATHROW);
   }
 
-  private void visitInvocationArguments(AbstractInvocation invocation) {
-    for (ExpressionStatement statement : invocation.getArguments()) {
-      statement.accept(this);
+  private List<String> visitInvocationArguments(AbstractInvocation invocation) {
+    List<String> argumentNames = new ArrayList<>();
+    for (ExpressionStatement argument : invocation.getArguments()) {
+      if (invocation instanceof FunctionInvocation && ((FunctionInvocation) invocation).usesNamedArguments()) {
```

Comment:
Have a look to the `java-interop`
https://github.com/golo-lang/golo-lang/pull/250/files#diff-73d861b2e4cdd14514d7975925c2b003R65

Basicaly if the bytecode contains the parameters info  it will work.